group/group.go

```diff
@@ -19,8 +19,15 @@ var (
 	P521 Group = wG{elliptic.P521()}
 )
 
+type Params struct {
+	ElementLength           uint // Length in bytes of an element.
+	CompressedElementLength uint // Length in bytes of a compressed element.
+	ScalarLength            uint // Length in bytes of a scalar.
+}
+
 // Group represents a prime-order group based on elliptic curves.
 type Group interface {
+	GetParams() Params
```

Comment:
Does it need to be called `GetParams`, instead of simply `Params`? Go's elliptic curve package has a nice pattern:

```
type Curve interface {
    // Params returns the parameters for the curve.
    Params() *CurveParams
   ...
```

Reply:
Updated now, and called `Params` to satisfy linter:
```
type name will be used as group.GroupParams by other packages, and that stutters; consider calling this Params
```

Reply:
Much better -- thanks!